Frontend/secrethive/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Remove the default `Readme` and add your readme for frontend folder, here you can add details about Frontend components.

Reply:
& you can add installation process here.